docs/integration-association.md

```diff
@@ -0,0 +1,37 @@
+# Association Integration
+## Desciption
+This file try to define how the associations will be handled in our application.
+Firstly, let see some user stories linked to associations. Then, how we can implement those functionalities :
+## User Stories
+- As a user interested in philosophy, I want to be able to find related associations so that I don't have to make too much effort.
+- As a user potentially interested in joining JDRPoly, I want to be able to get enough information on this association so that I know if it interests me or not.
+- As a curious user, I want to be able to discover new associations that I never thought of so that I can start a new activity.
+- As a user, I want to be able to follow an association so that I can easily see news and events related to it.
+- As a user in charge of communication in my association, I want to be able to share latest news so that people can see what's happening in my association.
+- As a user in charge of communication in my association, I want to be able to create events and certified them with my association so that other users can see that my event is from my association.
+## Implementation
+### Dataclass Association
+We already have a dataclass called Association. Currently, it has the following attributes : an "associationId", a "name" and a "description". We can add to this :
+- "relatedTags: Set<Tag>", the set of all tags related to the association
+- "moderators: Set<User>", the set of people that have the right to publish actuality, modify association description, add members and create events in the name of the association
+- TO DISCUSS : "members: Set<User>", list of all approved members of the association
+- "e-mail : String", a reference to the e-mail of the association if users want to contact them
+- "website: String? = "" ", a reference to the website of the association for those who have one
+- "instagram: String? = "" ", a reference to the instagram of the association for those who have one
```

Comment:
I'd make that a generic URL field (or a list of urls actually), rather than website and instagram specifically. Users might want to link to other things like other types of social media as well.